app/models/organization.rb

```diff
@@ -36,7 +36,11 @@ def all_assignments(with_invitations: false)
   end
 
   def github_client
-    token = users.limit(1).order("RANDOM()").pluck(:token)[0]
+    if Rails.env.test?
+      token = users.first.token unless users.first.nil?
```

Comment:
If we're test mode then we pick the true user token (which is the first user) instead of randomly picking

Reply:
:+1: